bspump/ipc/stream.py

```diff
@@ -40,9 +42,18 @@ async def outbound(self):
 		Handle outbound direction
 		'''
 		while True:
-			data = await self.OutboundQueue.get()
+
+			try:
+				data = await self.OutboundQueue.get()
+
+			except RuntimeError as e:
+				# Event loop has been closed during .get() call likely b/c of the application exit
+				L.log(asab.LOG_NOTICE, "Outbound queue for Stream has been closed: '{}'.".format(e))
```

Comment:
This must not be a NOTICE, especially when you are logging the exception.
Print WARNING please.

And I agree that this could be silent if the event loop is closed (the exception is wrong in a first place but we cannot do much with that).